test/powershell/Scripting/NativeExecution/NativeCommandArguments.Tests.ps1

```diff
@@ -0,0 +1,22 @@
+Describe "Native Command Arguments" -tags "CI" {
+    $testPath = 'TestDrive:\test.txt'
+    $powershellTestDir = $PSScriptRoot
+    while ($powershellTestDir -notmatch 'test[\\/]powershell$') {
+        $powershellTestDir = Split-Path $powershellTestDir
+    }
+    $echoArgs = Join-Path (Split-Path $powershellTestDir) tools/EchoArgs/bin/echoargs
+
+    It "Should handle quoted spaces correctly" {
+        $a = 'a"b c"d'
+        & $echoArgs $a 'a"b c"d' a"b c"d >$testPath
+        $testPath | Should Contain 'Arg 0 is <ab cd>'
+        $testPath | Should Contain 'Arg 1 is <ab cd>'
+        $testPath | Should Contain 'Arg 2 is <ab cd>'
+    }
+
+    It "Should handle spaces between escaped quotes" {
+        & $echoArgs 'a\"b c\"d' "a\`"b c\`"d" >$testPath
```

Comment:
I'd like to understand this better.

```
"a\`"b c\`"d"
```

There is a mix of two things:
backticks as powershell string escape mechanics and `\` as command line args escape mechanics

This is an equivalent of  `'a\"b c\"d'`
When I use ehcoargs from cmd:

```
EchoArgs.exe a\"b c\"d
```

I'm getting

Arg 0 is <a"b>
Arg 1 is <c"d>

So the behavior that is currently described with `Arg 1` doesn't seem right.